capplets/desktop-icons/mate-desktop-icons.pc

```diff
@@ -0,0 +1,10 @@
+prefix=/usr/local
```

Comment:
You can also remove the file mate-desktop-icons.pc.

Reply:
Done.